### PR TITLE
created people_table

### PR DESCRIPTION
**Pull request by avaevansm**

### CODE REVIEW

**Comment by clai88** [comment]
ship it squirrel

File: test/fixtures/people.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
```
Comment (MartinDally) [inline test/fixtures/people.yml]: can you edit these fixtures?

File: test/models/person_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class PersonTest < ActiveSupport::TestCase
```
Comment (MartinDally) [inline test/models/person_test.rb:3]: test that you can create and persist a person?

**Comment by avaevansm** [comment]
@MartinDally ready for review.

**Comment by MartinDally** [comment]
:shipit:

**Pull request merged**